plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/datatools/ui/wizard/DriverDefinitionWizard.java

```diff
@@ -54,10 +52,9 @@ public void create(DriverDefinition driverDefinition) {
 	 * @see org.jboss.reddeer.jface.wizard.WizardDialog#finish()
 	 */
 	@Override
-	public void finish() {		
-		new DefaultShell(WIZARD_TITLE);
```

Comment:
I use this wizard in hibernate tests and this line is redundant. 
Take a look at super implementation, which doesn't focus shell before clicking on finish and DriveDefinitionWizard is no special case. So I removed it to behave in the same way as other reddeer wizards